packages/statistics-module/src/index.js

```diff
@@ -1,8 +1,7 @@
-const { StatisticModel } = require("./model/index.js");
-const { StatisticUseCases } = require("./useCases/index.js");
+const { StatisticModel } = require('./entities');
+const { StatisticUseCases } = require('./useCases');
 
 module.exports = {
-  addStatistic: StatisticUseCases.addStatistic({
-    model: StatisticModel,
-  }),
-};
\ No newline at end of file
+  addStatistic: StatisticUseCases.addStatistic({ model: StatisticModel }),
+  getClicksByLinkId: StatisticUseCases.getClicksByLinkId({ model: StatisticModel }),
+};
```

Comment:
@calguzman We should export the getGlobalClicks method

```suggestion
  addStatistic: StatisticUseCases.addStatistic({ model: StatisticModel }),
  getClicksByLinkId: StatisticUseCases.getClicksByLinkId({ model: StatisticModel }),
  getGlobalClicks: StatisticUseCases.getGlobalClicks({ model: StatisticModel }),
};
```

Reply:
@calguzman This is the review